rero_ils/modules/acq_accounts/api.py

```diff
@@ -56,6 +56,17 @@ class AcqAccount(IlsRecord):
     minter = acq_account_id_minter
     fetcher = acq_account_id_fetcher
     provider = AcqAccountProvider
+    pids_exist_test = {
+        'required': {
+            'lib': 'library',
+            'budg': 'budget'
+        },
+        'not_required': {
+            'org': 'organisation',
+            'patr': 'patron'
```

Comment:
I do not think there is a link to the patron record here.

Reply:
You are right.